posthog/schema.py

```diff
@@ -53,6 +53,7 @@ class AlertState(StrEnum):
     FIRING = "Firing"
     NOT_FIRING = "Not firing"
     ERRORED = "Errored"
+    RESOLVED_MANUALLY = "Resolved manually"
```

Comment:
It's probably better if the string representations of these enums don't have spaces in them. It better represents them as machine readable, vs human readable.

Reply:
agreed, was about to do the same but went with convention here. This saves having an enum to human readable mapping. Will make sure to avoid for new enums